Autopilot/AttitudeManager/AttitudeManager.c

```diff
@@ -534,7 +534,7 @@ void imuCommunication(){
     VN100_SPI_GetYPR(0, &imuData[YAW], &imuData[PITCH], &imuData[ROLL]);
     imu_YawAngle = imuData[YAW];
     imu_PitchAngle = imuData[PITCH];
-    imu_RollAngle = (imuData[ROLL]);
+    imu_RollAngle = (imuData[ROLL] + 90);
```

Comment:
This is dangerous, as it changes the calculations in the PID loops. You want to change the orientation globally.
See line 222.